Documentation/README.md

```diff
@@ -12,6 +12,6 @@ Once you have a Rook cluster running, you can use the `rook` tool to create and
 - [Using Rook Guide](client.md)
 
 
-## Troubleshooting Toolbox
+## Advanced Configuration and Troubleshooting
```

Comment:
I'd say either remove Documentation/README.md or make it a brief index of the docs with a short description for each.  That whole file is a copy of what's already in the root README.md.